### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by AditiJalaj**

### CODE REVIEW

File: components/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      rel="shortcut icon"
+      href="../../assets/favicon.ico"
+      type="image/x-icon"
+    />
+    <link rel="stylesheet" href="../../index.css" />
+  </head>
+  <body>
+    <nav class="topnav">
+      <a class="link" href="/"
+        ><img class="header-image" src="../../assets/Ditsy-logo2.svg"
```
Comment (anuj-kumary) [inline components/alert/alert.html:18]: Give **alt** attribute to every image tag.

**Review by anuj-kumary — APPROVED** [review]
Looks good to merge!
- Suggestion:-  Always give a proper name to **alt** attribute it help for screenreader

**Review by shrey27 — APPROVED** [review]

**Review by codewithrishu — APPROVED** [review]
looks good to merge

**Comment by srijani07** [comment]
Looks good to merge. Suggestion: Add a screenshot in the PR description so that it becomes easier to review.

**Review by srijani07 — COMMENTED** [review]
Will be better to merge from badge to dev branch, instead of master. Also, add a screenshot in the PR description so that it becomes easier to review.

**Review by SadiqZak — APPROVED** [review]
Looks good to merge.

**Review by vaishnavipaliya11 — COMMENTED** [review]
Please don't make so many changes in a single PR.

**Pull request merged**